src/components/CurrentLocation.js

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+function CurrentLocation(props) {
+  const { location } = props;
```

Comment:
you can destructure this in the argument itself. instead of `props` just `{ location }`.